### ISSUE TITLE
Sprint 1.33.0 1/2 - Jan 24 to Feb 4

### ISSUE THREAD

**Issue by marcushyett-ph** [PostHog/posthog#8090]
# Global Sprint Planning

## Agenda
* Weekly reiterating of company goals for 2022
    * Goals:
      * Quality: Our core features (insights, recordings, feature-flags / experimentation) work like a Swiss watch
      * Extensibility: You can instantly connect PostHog to any other system 
      * Deployments: PostHog is the easiest self-hosted product to deploy and scale in the world
    * Non Goals:
        * Build lots of new low quality / partial features
        * Build everything a single enterprise customer wants just to close a deal
* Secondary on-calls and sentry focus @timgl

## Retro: Status of Outcomes from Previous Sprint

1. “Get 5 users running real experiments and sharing their experience” @neilkakkar Status: Terrible-ish. Great progress productwise, but its hard to get users running experiments quickly. They need longer lead times. (2/5, and the 2 are Paolo and moi)
2. Close out diagnosing causes and groups @EDsCODE Status: Paths/Funnels to Recordings query is done. Path query has been implemented by @rcmarron. Funnels coming up
3. Insights cleanup and scalability @macobo Status: Lifecycle speedup plus new groups features done. Some work got left over - cohorts cleanup. @EDsCODE : #7830 merged. Insight cohorts in. Or property filtering in progress
4. Make dashboards great again @Twixes Status: Not 100% guaranteed because of timing issues (spent 1 week as support hero, started this week ill, person page rework overrode priorities due to customer feedback), but I'm reasonably confident we'll have great dashboard by the end of the sprint
5. Collaboration: Taxonomy: Users should understand events actions and properties others have set up @alexkim205 Status: please update?
	  - https://github.com/PostHog/posthog/pull/8104
	  - https://github.com/PostHog/posthog/pull/8081
	  - In progress - mockups for contextual editing of taxonomies (event/event properties)
	  - In progress - events page revamp
	  - In progress - exposing definitions panel in appropriate places in the app (discovery of taxonomy)
6. Converge Helm chart and cloud: Support roll-forward and back for helm @fuziontech Status:
- https://github.com/PostHog/charts-clickhouse/pull/179 - is still getting worked on, but solid progress there
- https://github.com/PostHog/charts-clickhouse/issues/176 - Same story here ^^
- https://github.com/PostHog/charts-clickhouse/pull/243 - Some side task / customer support work was required 
- More details from @guidoiaquinti & @hazzadous who have been in the trenches ⛏️ 
7. Nail async migrations @tiina303 Status: WIP - improvements to system, more extensive testing, but haven't started testing with users yet

## Plan: Proposed Goals for Next Sprint 

Each goal should have a single owner. Owner can only be an engineer.

1. **Migration of cloud to helm chart**: Staging environment for cloud live! @guidoiaquinti + @hazzadous 
2. **Unblock Clickhouse upgrade:** 3 Key users with async migrations complete @tiina303 + @yakkomajuri 
3. **Performance**:~40% improvement for big clients: Events table migration on cloud complete @macobo 
2. **Validate Automated Insights Feasibility**: @EDsCODE (with @neilkakkar)
3. **Get customers using experiments**: @neilkakkar 
4. Insight/dashboards permissions (#7620) including permalink sharing with the project_id thing. Blocker for a large deal. @Twixes 
5. Exposing event and property taxonomy. @alexkim205 
6. Audit log. Heavily related to collaboration. We're going more into debt if we don't have it. @pauldambra 

# Team sprint planning

For your team sprint planning copy this template into a comment below for each team.

```
Team ___

## Retro

<!-- Talk about what went well, what didn't go well and any actions to improve next time -->

- 

## Hang over items from previous sprint

<!-- For each item, decide to re-prioritise (and add below) or deprioritise -->

- Item 1. prioritised/deprioritise

## Planning

<!-- Each item should have a single owner. Owner can only be an engineer. -->

### High priority

-

### Low priority / side quests

-

```

**Comment by mariusandra** [comment]
## Core Experience

### Retro

- @Twixes - It's been really productive, but not on the planned points 😜. Week doing support, second week started with an illness, and then worked on persons. Working on dashboards now, good progress by Friday, but not sure if nailed*2.
- @rcmarron - Was alright. Two directions: 1) recordings in insights - going good. 2) taxonomy stuff - not much progress. We have set direction, but no PRs. Recordings in insights work too much longer, as I did more work vs handing it over to Core Analytics for followup. It was great though, it's fun to be in those queries. Feeling more and more comfortable with the queries.
- @alexkim205 - Pretty productive. Pulled in a few directions, like leftover work for tooltips, etc. Mostly wrapped up now. With taxonomy, lots of scope setting & planning & backend code diving. Nothing visual to show yet. More taxonomy & collab in the next sprint.
- @pauldambra - Good. Wasn't feeling very productive, but got most of the datetime stuff, and wrapped up the APM stuff, so not terrible. Tried hard to do small PRs, did 3 large PRs. 
- @mariusandra - Back from holiday. Took a bit of time to ramp up. Worked on one tiny PR that became too huge for its own good. Good overall.
- @paolodamico - Edit and add.
- @clarkus - Edit and add.

### Hang over items

- Bit of everything for everyone: taxonomy, dashboards, datetimes. Read above.

### What do we want to work on?

- Audit log & edit history of objects (insights, flags, etc).... eventually leading to [HogFeed](https://github.com/PostHog/posthog/issues/8088) / social feed / activity log homepage 
- Meta team metrics --> a dashboard for how your team is using posthog.
- Revamping the events page. Work already in progress.
- Exposing that work (event & prop descriptions) in the taxonomic filters and other places...
- New demo system.
- Insights enchillada naming
- Finish datetime.
- Insight/dashboards permissions (#7620) including permalink sharing with the project_id thing. (Figma for inspiration) -- Blocker for a large deal.
- Improve client-side fuzzy search (it’s very daunting to search for “insight” as a property and see 20 less relevant results first)

### Priorities

- Insight/dashboards permissions (#7620) including permalink sharing with the project_id thing. Blocker for a large deal. @Twixes 
- Exposing event and property taxonomy. @alexkim205 
- Audit log. Heavily related to collaboration. We're going more into debt if we don't have it. @pauldambra

**Comment by fuziontech** [comment]
Team Platform 👾

## Retro

- @guidoiaquinti - Best sprint since he's joined!! 🎉  Basically having someone to have as a sparring partner between @hazzadous and @macobo in his timezone has made him feel a lot better about building and shipping. Not the most productive he's been, but the best he's felt during a sprint. Other things we can improve
- @hazzadous - It was good! He enjoyed working on helm and platform stuff. 
- @tiina303 - Wish the sprint was spent pairing up on async migrations but that plan changed due to priority changes, specifically around plugin server issues
- @fuziontech - Was a good sprint. We had fairly decent focus, but we can always get better. Cleaned up the board and made some solid progress on hiring. A long way to go still though.
- @yakkomajuri - **Plz fill in when you get a chance**

## Tickets and Priorities

Source of truth is our board https://github.com/orgs/PostHog/projects/10

**Comment by EDsCODE** [comment]
### Retro

**Experiments**: Last mile to beta release: Great progress, almost all things sorted out (significance is significant, distributions & exposure respected)
**Closing out groups**: Kind of went well - got a lot of new useful feedback though - all in the issue tracker.
**Closing out diagnosing causes**: Queries are ready; Path querying has been implemented by Rick; Funnel querying left to do
**Closing out insights**: Insight cohorting; Yeetcode step 1 in; Yeeted more viewsets; OR property filtering in progress; Lifecycle
done

**Support hero (Karl)**:
-_- enough that created https://github.com/PostHog/company-internal/issues/560
Really busy but low-productivity

### Planning:

**Validate Automated Insights Feasibility** — Eric
Automated Insights - https://github.com/PostHog/posthog/issues/8094

**Get customers using experiments** — Neil
Upselling experiments
Releasing content about experiments

**Cleanup**
Continued yeeting

Support Hero: Li
